flake.nix

```diff
@@ -0,0 +1,37 @@
+{
+  description = "Blockfrost API backend";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+
+  outputs = { self, nixpkgs }:
+    let
+      lib = nixpkgs.lib;
+      supportedSystems = [ "x86_64-linux" ]; # lib.systems.flakeExposed;
+      forAllSystems = f: lib.genAttrs supportedSystems (system: f system);
+      legacyPkgs = nixpkgs.legacyPackages;
+      default = lib.genAttrs supportedSystems (system: import ./default.nix {
+        inherit system;
+        pkgs = legacyPkgs.${system};
+      });
+    in
+    {
+      packages = forAllSystems (system: {
+        inherit (default.${system}) blockfrost-backend;
+        dockerImage = legacyPkgs.${system}.dockerTools.buildImage {
+          name = "blockfrost";
+          config = {
+            Cmd = [ "${self.packages.${system}.blockfrost-backend}/bin/blockfrost-backend" ];
+          };
+        };
+      });
+      # FIXME checks are not building
```

Comment:
I think it's safe for you to exclude these. They require disabled sandboxing as they access external database.

Reply:
Do you prefer that I completely remove them or we can leave them somewhere in the flake? Pperhaps conditionally only if the `--impure` flag is passed (never did something like this but should be definitely possible)